src/lib/tokenizer.ts

```diff
@@ -19,6 +19,7 @@ class Tokenizer {
   tokenize(): void {
     let tokenizedProgram = Tokenizer.program;
     tokenizedProgram = tokenizedProgram.replace(/\n/g, "_");
+    tokenizedProgram = tokenizedProgram.toLowerCase();
```

Comment:
I think a tricky issue with this that the user would lose their capital letters in their custom strings. How should we deal with this? One way would be to change the title to "goi - the all lowercase flash cards app" (kidding)

Reply:
Hmm right, let me figure something out